mixserv/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,7 @@
+# Set everything to be logged to the console
```

Comment:
log4j.properties in hivemall core should be used for mixserv. 

It might be a bug of shade-plugin description in the pom.xml of hivemall-core.
log4j.properties should be included in hivemall-core-xxx-with-dependencies.jar.